viewer/packages/pointclouds/wasm/src/parse_inputs.rs

```diff
@@ -64,11 +64,15 @@ pub fn parse_points(
     point_vec
 }
 
+const RADIUS_SCALE_FACTOR: f64 = 1.15;
```

Comment:
Also, maybe worth hard coding constants in typescript code, not in Rust and then passing values to the worker. Just for easier addition of possibly future features and also we should probably expose these values since frontend application that will write annotation data have to know which kind of adjustments are made for visual purposes and adjust for them if needed.

Reply:
But I guess you can decide whatever is easier for now.

Reply:
Yes, this is a good point... I'd rather not do that now, though.. Passing that value means getting it over the boundary to the worker and then over the boundary again to the wasm module :see_no_evil: I'll create a task for it